CHANGES.txt

```diff
@@ -18,6 +18,9 @@ Breaking Changes
 Changes
 =======
 
+ - Added support for hash sums (MD5, SHA1, SHA256). Please
+    see :ref:`sha1 <sha1>`.
```

Comment:
@mxm I'm really sorry but you can revert this since the CHANGES.txt becomes part of the release notes (inside generated html docs).